homeassistant/util/yaml.py

```diff
@@ -69,7 +69,7 @@ def load_yaml(fname: str) -> Union[List, Dict]:
             # We convert that to an empty dict
             return yaml.load(conf_file, Loader=SafeLineLoader) or OrderedDict()
     except yaml.YAMLError as exc:
-        _LOGGER.error(exc)
+        _LOGGER.error(str(exc))
```

Comment:
Exceptions have a `__str__` magic method so this shouldn't be needed.

Reply:
mypy complains that it is expecting str and got an Exception. Rather than add `# type: ignore` I think an explicit cast is more readable.

Reply:
I think mypy is wrong in this case. I'd prefer the ignore, but you decide, 🤷‍♂️.